README.md

```diff
@@ -32,4 +33,26 @@ docker run -d \
     nginx
 ```
 
+## Example of webhooks usage
+**Webhook config example**
+A webhook config must be valid JSON. If ```success``` is not specified in the config, git-sync will not wait for a response.
```

Comment:
If `success` is not specified, do we make any guarantees about behavior?  E.g. closing sockets?

If `success` is specified, how long will we wait for success?  We need a timeout or something.

Do we pass anything as body?

Do we need to allow multiple codes for success?  E.g. any 200?

Can the user assume this will follow 30{1,2,7,8} redirects properly?